### ISSUE TITLE
10.0.0 - TypeError: this.name is not a function

### ISSUE THREAD

**Issue by jdahlgren** [acacode/swagger-typescript-api#381]
Im getting the following error since the new version. The command worked in previous release:
Im running the command `          npx swagger-typescript-api -p filename.json -n outfile`

```
/home/vsts/.npm/_npx/ad2f6472e51cc62f/node_modules/commander/lib/command.js:1258
    const commandEvent = `command:${this.name()}`;
                                         ^

TypeError: this.name is not a function
    at Command._parseCommand (/home/vsts/.npm/_npx/ad2f6472e51cc62f/node_modules/commander/lib/command.js:1258:42)
    at Command.parse (/home/vsts/.npm/_npx/ad2f6472e51cc62f/node_modules/commander/lib/command.js:891:10)
    at Object.<anonymous> (/home/vsts/.npm/_npx/ad2f6472e51cc62f/node_modules/swagger-typescript-api/index.js:73:9)
    at Module._compile (node:internal/modules/cjs/loader:1105:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1159:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12)
    at node:internal/main/run_main_module:17:47
```

**Comment by janekx21** [comment]
+1

**Comment by e154** [comment]
+1

**Comment by Deadeye07** [comment]
+1

**Comment by js2me** [comment]
Seems that this problem based on closed issue in commander https://github.com/tj/commander.js/issues/1282

**Comment by js2me** [comment]
fixed in 10.0.1
